### DISCUSSION TITLE
improve bulk actions

### DISCUSSION THREAD

**Discussion by boudo2017** [Blair2004/NexoPOS#513] [Ideas]
Could you please improve this: you need to add : select All items
![Screenshot (4)](https://user-images.githubusercontent.com/22215705/149634370-061b81a6-fa22-4e4f-83a8-c8b5abb1aed6.png)
to be like this
![6t7TnP](https://user-images.githubusercontent.com/22215705/149634574-9c0f63a0-7fe3-433a-803f-e86d5fd353fb.png)
It is hard to delete page by page if we have 14 page like this example
regards

**Comment by Blair2004** [comment]
Hi, 
Why not. Thank you for the suggestion.
